pkgs/tools/backup/bupstash/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, rustc, cargo, ronn, pkg-config, libsodium }:
+
+stdenv.mkDerivation rec {
+  pname = "bupstash";
+  version = "0.6.3";
+
+  src = fetchurl {
+    url    = "https://github.com/andrewchambers/bupstash/releases/download/v${version}/bupstash-v${version}-src+deps.tar.gz";
+    sha256 = "0cvh50ar43wl0lbvck3bd315p6ffz7qspy31wk208p1anzk56x46";
+  };
+
+  buildInputs = [ rustc cargo ronn pkg-config libsodium ];
```

```suggestion
  nativeBuildInputs = [ ronn pkg-config ];
  buildInputs = [ libsodium ];
```

Please use buildRustPackage.